docs/markdown/snippets/prefer_pkgconf.md

```diff
@@ -0,0 +1,7 @@
+## Pkgconf is now the preferred pkg-config implementation
+
+Previous versions of meson would not look for `pkgconf` unless it had a
+`pkg-config` alias. Meson will now search for `pkgconf` first, as it is a
+superior implementation, and then for `pkg-config` second, if `pkgconf` is
+not found. For some OSes this will not result in any changes, as `pkg-config` is
```

Comment:
"For most OSes this will not result in any changes, as either pkgconf is not installed, or pkg-config is an alias to pkgconf."